### PR TITLE
Remove Girish Nair as a contact

### PR DESCRIPTION
**Pull request by BlueReZZ**

### CODE REVIEW

**Review by giorgiosironi — APPROVED** [review]
Any replacement?

**Comment by BlueReZZ** [comment]
> Any replacement?

Not yet - thought it was more important to remove him first. Finding a replacement, given the small size of the community, is difficult but I have someone in mind.

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 The community has elected to adopt the Contributor Convenant in order to foster an open and welcoming environment where contributors can feel safe to make contributions in any way they wish. 
 
 The full text of the code of conduct is below. Instances of abusive, harassing, or otherwise unacceptable behaviour may be
-reported by contacting one of our code of conduct representatives, David Moulton from eLife (d.moulton@elifesciences.org) or Girish Nair from Hindawi (girish.nair@hindawi.com).
+reported by contacting one of our code of conduct representatives, David Moulton from eLife (d.moulton@elifesciences.org).
```
Comment (davidcmoulton) [inline CODE_OF_CONDUCT.md]: Should "reported by contacting one of our code of conduct representatives" be changed to "reported by contacting our code of conduct representative" until a replacement for Girish is found?

Comment (BlueReZZ) [reply CODE_OF_CONDUCT.md]: Good spot, I've updated

**Review by davidcmoulton — APPROVED** [review]

**Review by NuclearRedeye — APPROVED** [review]

**Pull request merged**